tools/RestfulApiWebService/DB/createDB.sql

```diff
@@ -0,0 +1,19 @@
+USE [test]
```

Comment:
it is no need i will remove it

---
In reply to: [131037163](https://github.com/dotnet/docfx/pull/1906#discussion_r131037163) [](ancestors = 131037163)